### DISCUSSION TITLE
Strange log errors in version 107.26

### DISCUSSION THREAD

**Discussion by fizsef** [AdguardTeam/AdGuardHome#5596] [Q&A]
I began to observe such errors in the log in version 107.26 What is it, from where ...? There have never been such errors before.
![1](https://user-images.githubusercontent.com/74646605/225088668-0bac996f-d43d-43f7-8683-df21d61c2d37.jpg)

**Comment by ainar-g** [comment]
Seems like a bad response from Cloudflare.  `dig` agrees:

```sh
dig IN HTTPS 'yandex.ru' @1.1.1.3
```

```none
;; Warning: Message parser reports malformed message packet.

; <<>> DiG 9.18.1-1ubuntu1.3-Ubuntu <<>> IN HTTPS yandex.ru @1.1.1.3
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 23581
;; flags: qr rd ra; QUERY: 1, ANSWER: 1, AUTHORITY: 0, ADDITIONAL: 1
;; WARNING: Message has 38 extra bytes at end

;; QUESTION SECTION:
;yandex.ru.                     IN      HTTPS

;; Query time: 0 msec
;; SERVER: 1.1.1.3#53(1.1.1.3) (UDP)
;; WHEN: Tue Mar 14 20:45:44 MSK 2023
;; MSG SIZE  rcvd: 77
```

**Comment by fizsef** [comment]
At the same time, such a message - ";; Warning: Message parser reports malformed message packet." is issued only when a request to the domain is used yandex.ru , other domains do not have this message
